src/avram/associations.cr

```diff
@@ -56,9 +56,8 @@ module Avram::Associations
     define_has_one_base_query({{ assoc_name }}, {{ model }}, {{ foreign_key }})
   end
 
-  macro belongs_to(type_declaration)
+  macro belongs_to(type_declaration, foreign_key = nil, parent_table = nil)
```

Comment:
`parent_table` makes me think there's inheritance going on, but I don't believe that's how this works at all. I was also wondering about `table_for` or `table_name`. Sounds like the former might be reserved for something else?

Reply:
`table_for` is a macro to get the table name from a model class. `parent` is more like the belongs_to/has_one relationship. So Post (parent) and Comments (children). Maybe there is a better way to describe this though?

Reply:
Maybe we just call it `assoc_table`? Sometimes adding more words just makes things more confusing so maybe just `table`?

Reply:
I think I'll go with `table` for now, but you have a suggestion comment here. Happy to  change it if we find something great!